src/formatter/test_helpers.js

```diff
@@ -1,9 +1,69 @@
 import _ from 'lodash'
 import sinon from 'sinon'
+import Gherkin from 'gherkin'
+import Status from '../status'
 
 export function createMock(input) {
   if (_.isArray(input)) {
     input = _.zipObject(input)
   }
   return _.mapValues(input, value => sinon.stub().returns(value))
 }
+
+export function parseGherkinDocument(eventBroadcaster, data, uri, types, lang) {
```

Comment:
Nice. Can we update the parameters to use an options object, not include types as its not used, and rename lang to language? 

I'm thinking:
```js
parseGherkinDocument({eventBroadcaster, data, language, uri})
```